src/com/dotmarketing/portlets/browser/ajax/BrowserAjax.java

```diff
@@ -1143,24 +1177,24 @@ public boolean moveFile ( String inode, String folder ) throws Exception {
 
         try {
             Identifier ident=APILocator.getIdentifierAPI().findFromInode(inode);
-            IHTMLPage page = ident.getAssetType().equals("htmlpage")
+            IHTMLPage page = ident.getAssetType().equals("htmlpage") 
```

Comment:
The thing here is that a null coming from APILocator.getIdentifierAPI().findFromInode(inode) would be completely wrong, so, do we want to prolong the processing time with a wrong state (literal in left side would do so), which would eventually fail at some other point? or do we want to fail fast? I would vote for failing fast, so it might be better to checkNotNull for APILocator.getIdentifierAPI().findFromInode(inode)